src/etl/sars-cov-2/steps/add-vaccination-data-to-estimate-step.ts

```diff
@@ -157,6 +170,30 @@ export const addVaccinationDataToEstimateStep = (
     allStudies: input.allStudies,
     vaccinationData: input.vaccinationData,
     positiveCaseData: input.positiveCaseData,
+    countryPopulationData: input.countryPopulationData,
+    consolidatedCountryData: input.consolidatedCountryData.map((countryDataPoint) => ({
+      ...countryDataPoint,
+      countryPeopleVaccinatedPerHundred: getAssociatedVaccinationDataForEstimate({
+        estimate: {
+          threeLetterCountryCode: countryDataPoint.alphaThreeCode,
+          //Pick a mid date close enough to the middle of the month
+          samplingMidDate: new Date(countryDataPoint.year, countryDataPoint.month - 1, 14)
```

Comment:
minus 1 since the Date constructor zero indexes months but all the rest of our data one indexes months (understandably lol). 14 is close enough to the middle of the month to work for me :)